src/orders/views/OrderTransactionRefundEdit/OrderTransactionRefundEdit.tsx

```diff
@@ -65,6 +71,23 @@ const OrderTransactionRefund: React.FC<OrderTransactionRefundProps> = ({
     if (!data?.order || !draftRefund) {
       return;
     }
+    // @ts-expect-error - submit data actually returns a string
```

Comment:
so why not change that within: "OrderTransactionRefundPageFormData"? The type is wrong and can cause issues in the future, you can:
- Change type to string in OrderTransactionRefundPageFormData
- Change the type to string | number in OrderTransactionRefundPageFormData, then cover both cases in each place you use amount